data/tutorials/getting-started/01_tour.md

```diff
@@ -0,0 +1,752 @@
+---
+id: first-hour
+title: A Tour of OCaml
+description: >
+  Hop on the OCaml sightseeing bus. This absolute beginner tutorial will drive you through the marvels and wonders of OCaml. You'll have a sneak peek at everything that matters. Money back if you don't see OCaml's big five game.
```

Comment:
```suggestion
  Hop on the OCaml sightseeing bus. This absolute beginner tutorial will drive you through the marvels and wonders of OCaml. You'll have a sneak peek at everything that matters. Money back if you don't see OCaml's big five game.
```
"big five game" sounds a little strange to me... 
How about something like "OCaml's top game" or "OCaml's top five features / benefits"?

Reply:
No worries, this sentence will go away. https://en.wikipedia.org/wiki/Big_five_game